### DISCUSSION TITLE
Where are the settings located for Windows?

### DISCUSSION THREAD

**Discussion by kolbdog3333** [FlareSolverr/FlareSolverr#773] [Q&A]
For flaresolverr for windows where in the folder are the settings located to edit them so I can edit the settings so I can make one browser instance of chromium the main one.

**Comment by kolbdog3333** [comment]
And by settings I meant the environmental variables Where do I edit them in Windows

**Comment by ngosang** [comment]
You have to create a `.bat` file and run it instead of `flaresolverr.exe`
https://netlicensing.io/blog/2012/06/15/set-environment-variables-in-windows-shortcut/

**Reply by brunoventura22** [reply]
WINDOWS CHANGE ENVIROMENT VARIABLES

[EDIT FINAL] - Using an ELEVATED CMD and navegating to the directory did the trick. 

two years later and i'm here with the same problem. my bat file looks like this:

C:\Windows\System32\cmd.exe /c "SET HOST=127.0.0.1 && TZ=america/brasilia && LOG_LEVEL=debug && START /D ^"D:\Bruno Ventura\Downloads\Programas\qbit\flaresolverr^" flaresolverr.exe"

this host is the jacket host. I think that flaresolverr should serve on this host right?

opening a CMD/Shell, navigating to the flaresolverr folder and using set VARIABLE does not seem to do anything. No errors messages, but the variables do not change. I got to change timezone to mine, but just that.

**Reply by ilike2burnthing** [reply]
```bat
@ECHO OFF
SET HOST=127.0.0.1
SET TZ=America/Sao_Paulo
SET LOG_LEVEL=DEBUG
START /B "" "D:\Bruno Ventura\Downloads\Programas\qbit\flaresolverr\flaresolverr.exe"
```

I'm not sure `america/brasilia` is a valid TZ - https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

**Reply by brunoventura22** [reply]
> ```batchfile
> @ECHO OFF
> SET HOST=127.0.0.1
> SET TZ=America/Sao_Paulo
> SET LOG_LEVEL=DEBUG
> START /B "" "D:\Bruno Ventura\Downloads\Programas\qbit\flaresolverr\flaresolverr.exe"
> ```
> 
> I'm not sure `america/brasilia` is a valid TZ - https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Thank you! brasilia is the standard brazilian time, so my first thought was to use it. 

Now, only thing that I didn't figured yet is the /captcha directory. It doesn't exist by default right? And second, free captcha solvers are nowhere to be found too, is it correct? I tried just setting the EV CAPTCHA_SOLVER=hcaptcha-solver, but doesn't seem to make a difference. I'm getting the "ReqId 5864 The Cloudflare 'Verify you are human' button not found on the page." error. I'll read #1036, #1300, and #1163 as soon as I have some time o/

Thanks!

**Reply by ilike2burnthing** [reply]
Captcha solvers aren't currently supported - https://github.com/FlareSolverr/FlareSolverr#captcha-solvers

Current working fork (you'll have to run from source) - https://github.com/21hsmw/FlareSolverr
